src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -87,6 +137,13 @@ void IStreamsExecutor::Config::SetConfig(const std::string& key, const std::stri
         default:
             OPENVINO_UNREACHABLE("Unsupported affinity type");
         }
+        // correct unreasonable thread_binding_type entered by user
```

Comment:
Please only check and correct affinity between "CORES" and "HYBRID_AWARE" for hard pinning to right HW architecture. 

Skip this step if Input affinity is "NUMA"

Reply:
Ok, done.

Reply:
Move this correction to next PR